cmd/greeter_en/BUILD

```diff
@@ -1,6 +1,8 @@
 # Copyright 2021 Pants project contributors.
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
Total aside but: should we actually have these headers for files in the example repos?

Reply:
I'd be happy to get rid of them. @benjyw any reason you added them way back when?